### PR TITLE
Feature/cancel campaign advertisement

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
## Checklist:

- [ ] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (at least 80% coverage)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -49,7 +49,7 @@ ADVERTISEMENT_CONTRACT_NAME=advertisement
 #   Advertisement addresses
```
Comment (DioPires) [inline .env.example:49]: #### Tests:
```diff 
+ 112 passing
```
#### Test coverage: 
82.97 %

Comment (DioPires) [inline .env.example:49]: #### Tests:
```diff 
+ 112 passing
```
#### Test coverage: 
82.97 %

Comment (DioPires) [inline .env.example:49]: #### Tests:
```diff 
+ 112 passing
```
#### Test coverage: 
82.97 %

**Comment by GustavoRSSilva** [comment]
@tcrosado @zepcp pretty simple script to cancel an campaign.

**Review by tcrosado — APPROVED** [review]

**Pull request merged**